bucket/rawtherapee.json

```diff
@@ -0,0 +1,33 @@
+{
+  "homepage": "https://rawtherapee.com/",
+  "description": "RawTherapee is a free, cross-platform raw image processing program",
+  "license": "GPL v3",
+  "version": "5.7",
+  "url": "https://rawtherapee.com/shared/builds/windows/RawTherapee_5.7_WinVista_64.zip",
+  "hash": "f6d78951a5a398e259637ff8c3dadd5209bf6bde6694a3a3e168ba8bfd576316",
+  "installer": {
+    "script": "Expand-InnoArchive -Path \"$original_dir\\RawTherapee_5.7_WinVista_64.exe\" -DestinationPath \"$original_dir\""
+  },
+  "bin": [
```

Comment:
```suggestion
    "bin": [
        "rawtherapee-cli.exe",
         [
            "rawtherapee-cli.exe",
            "rawtherapee"
        ]
    ],
```

Reply:
according to https://rawpedia.rawtherapee.com/Command-Line_Options
rawtherapee and rawtherapee-cli are different

Reply:
Both have argument handler?

If yes, then use just `"bin": "["rawtherapee-cli.exe", "rawtherapee.exe"]` No need to rename shim with same name as original.

Reply:
yes both have it

updated
see 2a8fee2

Reply:
> Add hash extraction from https://rawtherapee.com/2019/09/repackaged-windows-build/

Reply:
I found sha256sum in https://rawtherapee.com/shared/builds/windows/

so i added autoupdate.hash

see dae95c8